[2025-04-18T14:46:24.896+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-04-18T14:46:24.917+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: data_pipeline.clean_data scheduled__2025-04-18T13:00:00+00:00 [queued]>
[2025-04-18T14:46:24.927+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: data_pipeline.clean_data scheduled__2025-04-18T13:00:00+00:00 [queued]>
[2025-04-18T14:46:24.928+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 2
[2025-04-18T14:46:24.946+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): clean_data> on 2025-04-18 13:00:00+00:00
[2025-04-18T14:46:24.953+0000] {standard_task_runner.py:72} INFO - Started process 2319 to run task
[2025-04-18T14:46:24.960+0000] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'data_pipeline', 'clean_data', 'scheduled__2025-04-18T13:00:00+00:00', '--job-id', '70', '--raw', '--subdir', 'DAGS_FOLDER/data_pipeline.py', '--cfg-path', '/tmp/tmp7_sz6j80']
[2025-04-18T14:46:24.964+0000] {standard_task_runner.py:105} INFO - Job 70: Subtask clean_data
[2025-04-18T14:46:24.988+0000] {logging_mixin.py:190} WARNING - /home/airflow/.local/lib/python3.12/site-packages/airflow/settings.py:209 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-04-18T14:46:25.025+0000] {task_command.py:467} INFO - Running <TaskInstance: data_pipeline.clean_data scheduled__2025-04-18T13:00:00+00:00 [running]> on host e8dafad5cd22
[2025-04-18T14:46:25.027+0000] {logging_mixin.py:190} WARNING - /home/airflow/.local/lib/python3.12/site-packages/airflow/settings.py:470 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-04-18T14:46:25.129+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='data_pipeline' AIRFLOW_CTX_TASK_ID='clean_data' AIRFLOW_CTX_EXECUTION_DATE='2025-04-18T13:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-04-18T13:00:00+00:00'
[2025-04-18T14:46:25.131+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-04-18T14:46:25.151+0000] {data_pipeline.py:51} INFO - Executing script: /opt/airflow/scripts/data_preprocessing.py
[2025-04-18T14:48:35.546+0000] {data_pipeline.py:53} INFO - Successfully executed: /opt/airflow/scripts/data_preprocessing.py
[2025-04-18T14:48:35.547+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-04-18T14:48:35.554+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-04-18T14:48:35.555+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=data_pipeline, task_id=clean_data, run_id=scheduled__2025-04-18T13:00:00+00:00, execution_date=20250418T130000, start_date=20250418T144624, end_date=20250418T144835
[2025-04-18T14:48:35.587+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-04-18T14:48:35.587+0000] {standard_task_runner.py:217} INFO - Process not found (most likely exited), stop collecting metrics
[2025-04-18T14:48:35.602+0000] {taskinstance.py:3901} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-04-18T14:48:35.606+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
